includes/class-wcpdf-main.php

```diff
@@ -634,9 +634,6 @@ public function maybe_reinstall_fonts( $force = false ) {
 			$this->copy_fonts( $fonts_path );
 
 			// save to cache
-			if ( get_transient( 'wpo_wcpdf_subfolder_fonts_has_files' ) !== false ) {
-				delete_transient( 'wpo_wcpdf_subfolder_fonts_has_files' );
-			}
 			set_transient( 'wpo_wcpdf_subfolder_fonts_has_files', 'yes' , DAY_IN_SECONDS );
```

Comment:
Where are we using this transient?

Reply:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/520-delete-font-transient-on-upgrade/includes/class-wcpdf-main.php#L575

Reply:
Thanks. It indeed looks like we are already doing everything we can to make sure the fonts are available. I think running a cron to reinstall the fonts daily, like one of the users suggested, is a bit overkill. And also doesn't address the issue in a correct way.